src/sql/parts/notebook/notebook.component.html

```diff
@@ -7,7 +7,7 @@
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div #toolbar class="editor-toolbar actionbar-container" style="flex: 0 0 auto; display: flex; flex-flow: row; width: 100%; align-items: center;">
 	</div>
-	<div class="scrollable" style="flex: 1 1 auto; position: relative" (click)="unselectActiveCell()">
+	<div #container class="scrollable" style="flex: 1 1 auto; position: relative" (click)="unselectActiveCell()" (scroll)="scrollHandler($event)">
```

Comment:
We have some (known) scroll issues in notebooks. Does this make those issues worse, better, or is there no change?

Reply:
Yes, it works.